webapp/src/components/ProfilePage/ProfilePage.container.js

```diff
@@ -22,11 +22,13 @@ const mapState = (state, { location, match }) => {
   let parcels = []
   let contributions = []
   let publishedParcels = []
+  let estates = []
   let mortgagedParcels = []
   if (address in addresses) {
     parcels = addresses[address].parcels
     contributions = addresses[address].contributions
     publishedParcels = addresses[address].publishedParcels
+    estates = addresses[address].estates.filter(e => e.data.parcels.length > 0)
```

Comment:
What do you think of moving this logic inside the `getAddreses` selector

Reply:
Nice! done 0174c65

Reply:
Nice! done e7d47b1